ee-9/common/pom.xml

```diff
@@ -626,6 +626,16 @@
 
         <!-- Misc -->
 
+        <dependency>
+            <groupId>org.antlr</groupId>
+            <artifactId>antlr4</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.antlr</groupId>
+            <artifactId>antlr4-runtime</artifactId>
+        </dependency>
```

Comment:
Please add an entry for this to https://github.com/wildfly/wildfly/blob/main/ee-9/common/src/main/resources/license/preview-feature-pack-common-licenses.xml

Reply:
Thanks Brian, entry added!